CHANGES/5389.bugfix

```diff
@@ -0,0 +1 @@
+Use pip-tools to pin dependencies
```

Comment:
It's not a bugfix, aiohttp-wise. It's probably misc or something like that.

Reply:
Done